src/Board.hpp

```diff
@@ -1,25 +1,37 @@
 #pragma once
+#include "FieldsEnum.hpp"
 #include "json.hpp"
 #include <vector>
+#include <iostream>
 
+using u_int = unsigned int;
```

Comment:
I dont see any advantage in using this alias, consider removing it.

Reply:
Ok, if it is not intuitive I am removing it.